<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestions</class>
 <widget class="QDialog" name="gestions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tarek</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>561</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Stock</string>
      </attribute>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>-152</x>
         <y>-42</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Produit du stock</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_qtestock">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Libelle</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_libstock">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantit√©</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date validation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prixstock">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouters">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbmodifierstock">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Produit au stock</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>0</y>
         <width>661</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string>Affichage des Stocks</string>
       </property>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tab_stock_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>351</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="recherchestock">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_date_diifrence">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>voir rest</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>270</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_day">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_d">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_moi">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_m">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_year">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_y">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="total">
        <property name="geometry">
         <rect>
          <x>99</x>
          <y>300</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_total">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>300</y>
          <width>74</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_total">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>300</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_dt">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>300</y>
          <width>74</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>240</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer produit du stock</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimers">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>160</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idstock">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_datestock">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Fournisseur</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom soc</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouterf">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nomsoc">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_lieu">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>lieu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_numero">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num√©ro</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type eq</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Fournisseur</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>0</y>
         <width>661</width>
         <height>521</height>
        </rect>
       </property>
       <property name="title">
        <string>Affichage des Fournisseur</string>
       </property>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tab_fournisseur_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>351</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>140</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_num_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pb_supprimerf">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>220</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pbmodifierfournisseur_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(0, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="le_typeeq">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>alimentation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>decoration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Statistique</string>
      </attribute>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>661</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="imp">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>imprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Mail</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>210</y>
         <width>228</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_55">
        <item>
         <widget class="QPushButton" name="sendBtn">
          <property name="text">
           <string>Envoyer les produit expirees</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>401</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_60">
          <property name="text">
           <string>destinataire √†
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rcpt"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>fathallah</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_11">
    <attribute name="title">
     <string>ahmed</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_13">
    <attribute name="title">
     <string>tahe</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_14">
    <attribute name="title">
     <string>fatma</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
